device/bgp/bgpupdatereplay.yaml

```diff
@@ -0,0 +1,31 @@
+components:
+  schemas:
+    Bgp.UpdateReplay:
+      description: >-
+        Ordered BGP Updates ( including both Advertise and Withdraws ) to be sent to the peer after the BGP session is established.
+      type: object
+      properties:
+        updates:
+          description: >-
+            Array of ordered BGP Updates ( including both Advertise and Withdraws ) to be sent to the peer after the BGP session is established. 
+          type: array
+          items:
+            $ref: '#/components/schemas/Bgp.OneUpdateReplay'
+          x-field-uid: 1
+    Bgp.OneUpdateReplay:
+      description: >-
+        Specification of one BGP Update to be sent to the BGP peer.        
+      type: object
+      properties:
+        update_bytes:    
+          description: >-
+            Bytes to be sent to peer after the BGP Update Header. The Update Header will always have the initial 16 bytes containing 
+            Marker bytes, 2 bytes containing the Length and 1 byte containing the Type.The string MUST contain sequence of valid hex bytes.
+            The bytes specified should be appended to  the Update message to be sent to the peer after the fixed 19 bytes above. 
+            The maximum length of 4077 is obtained from the maximum length allowed for a BGP message in RFC4271 which is 4096 minus mandatory 
+            19 bytes described above.
```

Comment:
Will suggest to add the following sentence :- 
"This byte stream can be of any length from <Apratim, give the min length> to 4077"

Reply:
Done,